.mochaTsMixin.js

```diff
@@ -0,0 +1,3 @@
+require("ts-node").register({
+  project: "test/tsconfig.json",
+});
```

Comment:
I cannot see why the name starts with `.` and the file is placed in the root directory.
`test/ts-node-register.js` makes more sense to me.

Reply:
I wonder if it needs the js extension...

I have gitignore and clean set to test/**/*.js

Reply:
I see.
How about adding `!test/ts-node-register.js` to .gitignore or create a `test/dist` directory for build output?

Reply:
It works without any extensions, so I have renamed it.

Reply:
It may work, but I don't think it's a good solution because newcomers will wonder why this file does not have an extension.